pyaedt/icepak.py

```diff
@@ -4277,9 +4277,10 @@ def assign_free_opening(
         pressure : float or str or dict, optional
             Prescribed pressure (static or total coherently with flow type) at the boundary. If a
             string is set, a variable name or a number with the unit is expected. If a float is set,
-            the unit ``'Pa'`` is automatically added. Also, a dictionary containing the keys ``'Function'``
-            and ``'Values'`` can be passed to set a transient behaviour. The acceptable values associated
-            with those keys can be found in the Icepak documentation. Default is ``"AmbientPressure"``.
+            the unit ``'pascal'`` is automatically added. Also, a dictionary containing the keys
+            ``'Function'`` and ``'Values'`` can be passed to set a transient behaviour. The acceptable
```

Comment:
```suggestion
            ``'Function'`` and ``'Values'`` can be passed to set a transient behavior. The acceptable
```
We use American rather than British spellings. Sorry!